src/openexr.imageio/exrinput.cpp

```diff
@@ -319,7 +320,9 @@ void set_exr_threads ()
 
     int oiio_threads = 1;
     OIIO::getattribute ("exr_threads", oiio_threads);
-
+    if (oiio_threads == 0) {
+        oiio_threads = Sysutil::hardware_concurrency();
```

Comment:
I never actually profiled testing *virtual* vs. *physical*, but for instance here I have 2 Xeon with 6 hyper threaded core each, which means overall 24 threads. Using physical would only ever use half of the processor, and would never exceed 1200% in **htop**. 

I'm not sure you need to worry about *over-threading*, basically as long as the processor is capped, this is because it is doing something useful: everything that OpenEXR is doing with these threads is uncompressing data.

 I think it's then the responsibility of the application using OIIO to actually handle this correctly, OIIO should just allow full access to all threads.